packages/candy_editor/src/assets/formTypes/IntForms/integersForm.tsx

```diff
@@ -163,31 +181,41 @@ export const IntegersForm = (editor: CandyClassEditor) => {
           {editor.property.immutable ? (
             <>
               <Grid column={1}>
```

Comment:
Grid column 1 is identical in both cases, so it should be moved above line 181.
<img width="966" alt="Screenshot 2023-02-14 at 5 02 16 PM" src="https://user-images.githubusercontent.com/94973362/218890876-023061bd-992e-495e-8dda-57e55529745c.png">

Reply:
moved above in all components.

Reply:
moved grid columns 1 & 2 above the ternary operator in all the components